Core/src/org/sleuthkit/autopsy/texttranslation/translators/GoogleTranslator.java

```diff
@@ -59,26 +60,31 @@ public GoogleTranslator() {
         settingsPanel = new GoogleTranslatorSettingsPanel(settings.getCredentialPath(), settings.getTargetLanguageCode());
         loadTranslator();
     }
-    
+
+    /**
+     * Check if google is able to be reached
+     *
+     * @return true if it can be, false otherwise
+     */
     private static boolean googleIsReachable() {
         String host = "www.google.com";
         InetAddress address;
         try {
             address = InetAddress.getByName(host);
             return address.isReachable(1500);
-        }catch (UnknownHostException ex) {
+        } catch (UnknownHostException ex) {
             return false;
```

Comment:
Shouldn't we log these exceptions? They may provide useful debug info.

Reply:
I doubt them providing any useful information, but that is because I doubt this being a useful check in general, I will log them though.